vscode/src/custom-prompts/utils/menu.ts

```diff
@@ -10,15 +11,13 @@ export const NewCustomCommandConfigMenuOptions = {
 }
 
 const inlineSeparator: QuickPickItem = { kind: -1, label: 'inline' }
-const chatOption: QuickPickItem = { label: 'Ask a Question' }
-const fixOption: QuickPickItem = { label: 'Refactor This Code' }
+const chatOption: QuickPickItem = { label: '/ask', description: 'Ask a Question', alwaysShow: true }
+const fixOption: QuickPickItem = { label: '/fix', description: 'Refactor This Code', alwaysShow: true }
```

Comment:
```suggestion
const fixOption: QuickPickItem = { label: '/fix', description: 'Refactor Selected Code', alwaysShow: true }
```

I found `This` not clear and `Selected` reminds users this command requires code to be selected --wdyt?

Reply:
Agree! Addressed in https://github.com/sourcegraph/cody/pull/798/commits/d624d32042b734da856bdbcfa5bff995efad9d01